frontend/src/concepts/pipelines/content/createRun/contentSections/ParamsSection/ParamsSection.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from 'react';
+import { Alert, FormGroup, FormSection, TextInput } from '@patternfly/react-core';
+import {
+  CreateRunPageSections,
+  runPageSectionTitles,
+} from '~/concepts/pipelines/content/createRun/const';
+import {
+  InputDefParamType,
+  PipelineVersionKFv2,
+  RuntimeConfigParameters,
+} from '~/concepts/pipelines/kfTypes';
+import { getInputDefinitionParams } from '~/concepts/pipelines/content/createRun/utils';
+import { IntegerInputParam } from './IntegerInputParam';
+import { RadioInputParam } from './RadioInputParam';
+import { JsonInputParam } from './JsonInputParam';
+
+type ParamsSectionProps = {
+  runParams: RuntimeConfigParameters | undefined;
+  version: PipelineVersionKFv2 | null;
+  onChange: (params: RuntimeConfigParameters) => void;
+};
+
+export const ParamsSection: React.FC<ParamsSectionProps> = ({
+  runParams = {},
+  version,
+  onChange,
+}) => {
+  const inputDefinitionParams = getInputDefinitionParams(version);
+
+  const renderContent = (): React.ReactNode => {
+    if (!version) {
+      return (
+        <Alert
+          variant="info"
+          isInline
+          isPlain
+          title="You must select a pipeline and version before you can set parameters."
+        />
+      );
+    }
+
+    if (!Object.keys(runParams).length) {
+      return <Alert variant="info" isInline isPlain title="This pipeline has no parameters." />;
+    }
+
+    return Object.entries(runParams).map(([label, value]) => {
+      const parameterType = inputDefinitionParams?.[label].parameterType;
+      const inputProps = {
+        value,
+        id: label,
+        name: label,
+        onChange: (_event: React.ChangeEvent<unknown>, newValue: string | number | boolean) =>
+          onChange({ ...runParams, [label]: newValue }),
+      };
+      let input: React.ReactNode;
+
+      switch (parameterType) {
+        case InputDefParamType.NumberInteger:
+          input = <IntegerInputParam {...inputProps} />;
+          break;
+        case InputDefParamType.Boolean:
+          input = <RadioInputParam {...inputProps} value={String(value)} />;
+          break;
+        case InputDefParamType.List:
+        case InputDefParamType.Struct:
+          input = <JsonInputParam {...inputProps} />;
+          break;
+        case InputDefParamType.NumberDouble:
+          input = (
+            <TextInput
```

Comment:
So the mocks ask for validation here... i dont know if that needed if we set the type to number. however the min should not be 0

Reply:
Why should the min not be 0?

Reply:
a pipeline run and KF accepts non positive numbers as input, so we should not restrict them.